src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -60,14 +60,21 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
 }) => {
   const styles = useStyles();
   const {t} = useTranslation();
-  const {deviceIsInspectable} = useMobileTokenContextState();
+  const {
+    deviceIsInspectable,
+    isError: mobileTokenError,
+    fallbackEnabled,
+  } = useMobileTokenContextState();
   const tokensEnabled = useHasEnabledMobileToken();
   const {customerProfile} = useTicketState();
   const hasProfileTravelCard = !!customerProfile?.travelcard;
 
   const showProfileTravelcardWarning = !tokensEnabled && hasProfileTravelCard;
 
-  const showNotInspectableTokenWarning = tokensEnabled && !deviceIsInspectable;
+  const showInspectableTokenWarning =
```

Comment:
could this be simplified to **tokensEnabled && (mobileTokenError ? !fallbackEnabled : !deviceIsInspectable);** or something?